operator/duties/validatorregistration.go

```diff
@@ -0,0 +1,70 @@
+package duties
+
+import (
+	"context"
+
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	spectypes "github.com/bloxapp/ssv-spec/types"
+	"go.uber.org/zap"
+
+	"github.com/bloxapp/ssv/logging/fields"
+)
+
+const validatorRegistrationEpochInterval = uint64(10)
+
+type ValidatorRegistrationHandler struct {
+	baseHandler
+
+	validatorsPassedFirstRegistration map[string]struct{}
```

Comment:
Do we need a mutex or https://github.com/cornelk/hashmap ?

Reply:
no need, there no parallel access to the map